app/client/src/pages/aboutus.tsx

```diff
@@ -1,52 +1,111 @@
 import * as React from 'react';
-import { Paper, Grid, Typography } from '@material-ui/core';
+import Image from 'next/image';
+import { Grid, Typography, Hidden } from '@material-ui/core';
 import { makeStyles } from '@material-ui/core/styles';
 
 const useStyles = makeStyles((theme) => ({
     root: {
         width: '100%',
-        height: '100%',
+        minHeight: '85vh',
+        paddingLeft: '1rem',
     },
-    paper: {
-        padding: theme.spacing(3),
+    section: {
+        display: 'flex',
+        flexDirection: 'column',
+        alignItems: 'center',
+        gap: '1rem',
+        textAlign: 'center',
+    },
+    paragraph: {
+        fontSize: 'large'
     },
 }));
 
-const AboutUs = () => {
+export default function AboutUs() {
```

Comment:
Nitpicky comment: I've been trying to move the project away from using default exports.  Please remove `default`.